examples/pylab_examples/annotation_demo.py

```diff
@@ -91,7 +91,7 @@
 fig = plt.figure()
 ax = fig.add_subplot(111, projection='polar')
 r = np.arange(0, 1, 0.001)
-theta = 2*2*np.pi*r
+theta = 4 * np.pi * r
```

Comment:
no, @efiring explained that one correctly.  If it isn't faster, use the more obvious way to write it.